RecoLocalCalo/HGCalRecAlgos/src/HGCal3DClustering.cc

```diff
@@ -102,9 +102,13 @@ std::vector<reco::HGCalMultiCluster> HGCal3DClustering::makeClusters(const reco:
 
       }
       if( temp.size() > minClusters ) {
+	math::XYZPoint position = clusterTools->getMultiClusterPosition(temp);
+	if (abs(position.z()) < 100) continue;
```

Comment:
It's an arbitrary cut, in principle `getMultiClusterPosition` returns (0., 0., 0.), see https://github.com/clelange/cmssw/blob/3c144cb9c5be207d4c2205bb8d8fd36d5d90176c/RecoLocalCalo/HGCalRecAlgos/src/ClusterTools.cc#L86-L88 - do you have a good check for that? I can also check (0., 0., 0.), but all that matters is that the multicluster z is outside the HGCal.